wsl-module/wsl.py

```diff
@@ -268,12 +267,21 @@ def main():
         # else:
         #     G_LOGGER.error('RPC Message failed to node_address')
 
-        # Sampling inter-message times from a Poisson distribution)
-        next_time_to_msg = poisson_interval(config['general']['msg_rate'])
+        # Compute the time to next message
+        # NOTE: Shall we pack this into a function?
```

Comment:
Take into account that you should try to split into functions as much as possible. This will help you a lot when you do tests. The shortest the function, the best for testing. Try to follow single responsibility principle when possible.

I would leave as it is now, but take that into account when adding tests, if not, big functions will be very problematic and messy to test.

Reply:
Added get_next_time_to_msg() to clean up a bit